src/js/contentScript.js

```diff
@@ -90,4 +122,25 @@ const domInspector = (function ()
     return _inspectElements();
 });
 
-domInspector();
\ No newline at end of file
+/*
+receive request from plugin to power on or power off and hide or show hate speech
+*/
+chrome.runtime.onMessage.addListener(function(request, sender, sendResponse){
+     if(request.command === SWITCH_OFF){
+         _revertElements();
+     }else if (request.command === SWITCH_ON) {
+        domInspector();
```

Comment:
When you have multiple tabs with blocked words open and turn off the plugin in one, the other pages don't get reverted back. I think it might be solved by adding a listener on active tabs, like when I do for updateToolbarIcon. Everytime the focus tab changes, check the power flag and call either inspector or revert